dataflow/src/main/java/org/checkerframework/dataflow/analysis/AbstractAnalysis.java

```diff
@@ -200,6 +200,14 @@ public AnalysisResult<V, S> getResult() {
             assert !n.isLValue() : "Did not expect an lvalue, but got " + n;
             if (!currentNode.getOperands().contains(n)
                     && !currentNode.getTransitiveOperands().contains(n)) {
+                if (n instanceof AssignmentNode) {
```

Comment:
Please add some comments that explain this block.
Should it be special-cased to only apply to assignments from post-fix operations?